src/authorizations/components/EditTokenOverlay.tsx

```diff
@@ -99,7 +93,13 @@ const EditTokenOverlay: FC<Props> = props => {
     }
 
     setPermissions(newPerms)
-  }
+  }, [props])
+
```

Comment:
I think adding `props` as a dependency in a useEffect will create a new warning (and creates loop risk) - so I would consider destructuring this first, before adding it to the array. [Adding props as dependency in a useEffect](https://stackoverflow.com/questions/72757679/react-adding-props-something-as-a-dependency-of-useeffect)